Module/uti/abdamaryllia.uti.json

```diff
@@ -2,7 +2,7 @@
   "__data_type": "UTI ",
   "AddCost": {
     "type": "dword",
-    "value": 25
+    "value": 250
```

Comment:
Abdamaryllia isn't a tier 5 item so I'm confused why it's priced as one based on your chart? Can you explain what I'm missing?

Reply:
"All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at."

40% of 250 is 100, which is the correct buying price for a tier 2 loot item according to the table above that you okay'd. IE: the player will only receive 100 credits for selling this item.

Reply:
The 40% was what I was missing. Thanks